flight/PiOS/STM32F4xx/pios_can.c

```diff
@@ -222,8 +222,56 @@ static void PIOS_CAN_RegisterTxCallback(uintptr_t can_id, pios_com_callback tx_o
 //! The mapping of message types to CAN BUS StdID
 static uint32_t pios_can_message_stdid[PIOS_CAN_LAST] = {
 	[PIOS_CAN_GIMBAL] = 0x130,
+	[PIOS_CAN_ATTITUDE_ROLL_PITCH] = 0x185,
+	[PIOS_CAN_ATTITUDE_YAW] = 0x196,
+	[PIOS_CAN_BATTERY_VOLT] = 0x2A0,
+	[PIOS_CAN_BATTERY_CURR] = 0x2A1,
+	[PIOS_CAN_RSSI] = 0x2A2,
+	[PIOS_CAN_ALT] = 0x2A3,
+	[PIOS_CAN_FLIGHTSTATUS] = 0x2A4,
+	[PIOS_CAN_GPS_LATLON] = 0x2B1,
+	[PIOS_CAN_GPS_ALTSPEED] = 0x2B2,
+	[PIOS_CAN_GPS_FIX] = 0x2B3,
+	[PIOS_CAN_GPS_VEL] = 0x2B4,
+	[PIOS_CAN_POS] = 0x2B5,
+	[PIOS_CAN_ALARM] = 0x2B6,
```

Comment:
Good idea. This is definitely meant to be common between F3/F4 definitions. Thanks for catching that.